src/data-dictionary/events/K8sNodeSample/condition.{conditionName}={conditionValue}.md

```diff
@@ -0,0 +1,11 @@
+---
+name: condition.{conditionName}={conditionValue}
```

Comment:
I wonder if we take a similar format for this as we did with Label, and do 

condition.CONDITION_NAME

and just leave it at that. I think 'conditionValue' is kind of implied. They'll all have values.

Reply:
@zuluecho9 I'm just going to leave this alone. We can update this later, if necessary.